job.go

```diff
@@ -133,3 +172,37 @@ func runJobs() {
 		setConfig("completed_builds", builds+1)
 	}
 }
+
+func runTestRunnerJobs() {
+	for {
+		job := <-reqTestRunnerch
+		log.Printf("runjob: %v\n", job.String())
+		setCurrentJob(&job)
+		repo, xmlFile, nodeCount := job.Repo, job.BuildXML, job.NodeCount
+		iniFile, confFile := job.IniFile, job.ConfFile
+
+		cmd := exec.Command(
+			"./runTests.sh",
+			osBuildMapping["centos6"],
+			repo,
+			xmlFile,
+			strconv.Itoa(nodeCount),
+			iniFile,
+			confFile)
+
+		cmdOutput := &bytes.Buffer{}
+		cmd.Stdout = cmdOutput
+		cmd.Stderr = cmdOutput
+
+		if err := cmd.Start(); err != nil {
+			log.Printf("cmd.Start(): %v\n", err)
+		}
+		cmd.Wait()
```

Comment:
can we log at the client, the zipped cb-collect files in s3 ? that would be useful.